docs/graphcast/radios/poi-radio.md

```diff
@@ -145,9 +161,84 @@ RUST_LOG="warn,hyper=warn,graphcast_sdk=debug,poi_radio=debug"
 
 ## Monitoring the Radio
 
-If the Radio operator has set up a Slack Bot integration and the Radio finds a POI mismatch, it sends alerts to the designated channel. The operator can also inspect the logs to see if the Radio is functioning properly, if it's sending and receiving messages, if it's comparing normalised POIs, if there is a found POI mismatch, etc.
+### Notifications
+
+If the Radio operator has set up a Slack Bot and/or Discord bot integration and the Radio finds a POI mismatch, it sends alerts to the designated channels. The operator can also inspect the logs to see if the Radio is functioning properly, if it's sending and receiving messages, if it's comparing normalised POIs, if there is a found POI mismatch, etc.
+
+### Prometheus & Grafana
+
+The POI Radio exposes metrics that can then be scraped by a Prometheus server and displayed in Grafana. In order to use them you have to have a local Prometheus server running and scraping metrics on the provided port. You can specify the metrics host and port by using the environment variables `METRICS_PORT` and `METRICS_HOST`. We also provide a [Grafana dashboard config JSON file](https://github.com/graphops/poi-radio/blob/main/poi-radio-grafana.json) which you can use to visualise the metrics in Grafana.
+
+## HTTP Server
+
+If you set the `SERVER_HOST` and `SERVER_PORT` environment variables, the Radio will spin up an HTTP server with a GraphQL API. Currently the Radio supports the following routes:
+
+- `/health` for health status
+- `/metrics` for http service metrics about the server
+- `/api/v1/graphql` for GET and POST requests with GraphQL playground interface
+
+Here are some example queries you can use:
+
+```graphql
```

Comment:
The below block is GraphQL schema, not example queries. 

If you think it is better to include example queries instead, this might be better
```
Query {
  radioPayloadMessages{
    identifier
    nonce
    blockNumber
    network
    signature
  }
  localAttestations{
    deployment
    blockNumber
    attestation{
      npoi
    }
  }
  comparisonResults(deployment:"Qm...."){
    deployment
    blockNumber
    resultType
    localAttestation{
      npoi
    }
    attestations{
      npoi
    }
  }
}
```